### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (h1)

### ISSUE THREAD

**Issue by animania4ka** [ada-cat/accessibility#8]
**Issue**: `Elements must have sufficient color contrast` ([`color-contrast`](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))

**Target application**: [Ishinryu Karate SF](https://www.ishinryukaratesf.com/)

**Element path**: #yui_3_17_2_1_1556832656296_286 > h1

**Snippet**:

    <h1 id="sqs-slash-page-header" style="display: inline-block;">Where the art and the sport meet</h1>

**How to fix**:

    Fix any of the following:
      Element has insufficient color contrast of 2.84 (foreground color: #999999, background color: #ffffff, font size: 12.0pt, font weight: bold). Expected contrast ratio of 4.5:1

**Environment**:
Chrome version 75.0.3770.0

====

This accessibility issue was found using Accessibility Insights for Web 2.1.2 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
